com.google.AndroidStudio.json

```diff
@@ -87,8 +87,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.28.0.tar.xz",
-                    "sha256": "dfa5d1a253aa451465478fe45c6a40ab8605b340fdb4c4e80b16d7f87708439d"
+                    "url": "https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.9.5.tar.xz",
```

Comment:
Are you rolling back Git binary from 2.28.0 to 2.9.5?

Reply:
It was a mistake